### ISSUE TITLE
Byssan full lyrics missing repeating part

### ISSUE THREAD

**Issue by Cactooz** [insektionen/songlist#14]
The song "Byssan full" seems to be missing //: :// around the first 3 lines compared to the real song "Byssan lull" by Evert Taube and other snapsvisor variations of that song.
```
//: Byssan lull, utav brännvin blir man full,
och slipsen man doppar i smöret. ://
Och näsan den blir röd,
och ögonen får glöd,
men tusan så bra blir humöret!
```
Instead of the current
```
Byssan lull,
utav brännvin blir man full,
och slipsen man doppar i smöret.
Och näsan den blir röd,
och ögonen får glöd,
men tusan så bra blir humöret!
```
It seems more fitting to add the //: :// so it follows the real song, since the song it so short it shouldn't really be a problem either imho.

**Comment by StelFoog** [comment]
Yes, I think that makes sense. Add it to the PR!

**Comment by Cactooz** [comment]
Resolved by #13
